### PR TITLE
Added DefCon under CyberSecurity

### PR DESCRIPTION
**Pull request by Sabyasachi-Seal**
## Adding a conference
- [x] Suggestion is not a duplicate
- [x] Suggestion is under the proper category
- [x] Link to conference is included 
- [x] Name of conference is included
- [x] Name and link are spelled correctly

## Adding a category
- [x] Category is not a duplicate
- [x] Category is spelled properly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,3 +127,11 @@ DevOps Stage is one of the biggest conferences on Cloud Architecture and DevOps
 This international conference brings together UI/UX engineers from all regions to share ideas, experiences, and tools.<br>
 **City and State**: Budapest<br>
 **Country**: Hungary
+
+## CyberSecurity
+
+[DefCon](https://defcon.org/)
+
+DEF CON is the world's premiere Hacker Conference, held annually in Las Vegas, Nevada. DEF CON is one of the oldest continuously running hacker conventions around, and also one of the largest.
```
Comment (jdwilkin4) [inline README.md:135]:
```suggestion
DEF CON is the world's premiere Hacker Conference, held annually in Las Vegas, Nevada. DEF CON is one of the oldest continuously running hacker conventions around, and also one of the largest.<br>
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by Sabyasachi-Seal** [comment]
Hey @jdwilkin4 , I have made the changes as requested.
Thanks for the help :)

**Review by jdwilkin4 — APPROVED** [review]
Hi @Sabyasachi-Seal ! 

Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by Sabyasachi-Seal** [comment]
Closes #24

**Pull request merged**